src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -94,6 +92,12 @@ public void run() {
                 BossBar.delete();
             }
             time--;
+        } else if (GameHandler.getGameHandler().getMatch().getState().equals(MatchState.WAITING)) {
+            if (neededPlayers() > 0) {
+                BossBar.sendGlobalBossBar(waitingPlayerMessage(), 100F);
+            } else {
+                BossBar.delete();
```

Comment:
I don't think we need automatic start, if both teams /ready, it will start
maybe i should make it so that if the /ready the countdown doesn't start if  all teams don't bypass the min tag